app/src/main/java/com/glowstudio/android/blindsjn/feature/board/viewmodel/PostViewModel.kt

```
@@ -116,4 +116,24 @@ class PostViewModel : ViewModel() {
+                val request = LikePostRequest(post_id = postId, user_id = userId)
+                val response = PostRepository.likePost(request)
+                if (response.isSuccessful) {
+                    // 서버에서 최신 게시글 정보 다시 불러오기
+                    val updatedPost = PostRepository.loadPostById(postId).body()?.data
+                    loadPostById(postId)
+                    loadPosts()
+                    onResult(true, updatedPost?.isLiked ?: false, updatedPost?.likeCount ?: 0)
+                } else {
+                    onResult(false, _selectedPost.value?.isLiked ?: false, _selectedPost.value?.likeCount ?: 0)
+                }
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)